src/components/Dropdown/V2/Dropdown.css.js

```diff
@@ -147,6 +147,15 @@ export const ItemUI = styled('div')`
     }
   }
 
+  &.c-SelectionClearerItem {
+    border-bottom: 1px solid ${getColor('grey.600')};
```

Comment:
![Screen Recording 2019-04-18 at 11 32 AM](https://user-images.githubusercontent.com/2322354/56372812-bb4bae80-61cd-11e9-8c61-ac282a878f36.gif)

@tinkertrain UI wise, is there a way to add the `Divider` component in between the clearer + list of items? The reason is because it's already styled with the desired margins :).

If not, would we be able to update these styles to look the same? Specfically margin/padding:

![Screen Shot 2019-04-18 at 11 32 13 AM](https://user-images.githubusercontent.com/2322354/56372885-de765e00-61cd-11e9-9f21-25afa5da7f36.png)

(Notice the selected BG is touching the divider line)

Thanks!! <3

Reply:
Done!